### DISCUSSION TITLE
Where to store config files for containers/pods

### DISCUSSION THREAD

**Discussion by nine-2-five** [microsoft/aspire#6983] [Q&A] [answered]
I'm currently ping-ponging around available options for service config files storage (tempo.yaml, grafana.ini...). What would be the most compatible way (local to k8s) of handling storage for these files? It seems quite a bit of work to mount a volume to dummy container just to copy a couple of files locally, and probably the same thing goes for k8s (times the number of nodes).
I couldn't find any mention of ConfigMaps in Aspire docs.

**Comment by davidfowl** [comment]
Are you asking about local development?

**Reply by nine-2-five** [reply]
Both, if possible. I'm looking for a solution with minimal changes between local and prod environment. I have S3 storage if that is supported in any way in both envs.

**Reply by davidfowl** [reply, marked as answer]
I'm not sure it's possible to accomplish that today. There's no abstraction that will mounted volumes in both cases in a consistent manner.
